### PR TITLE
Production model from asset

### PR DESCRIPTION
**Pull request by katrilh**
## Description
Please describe the change you have made.

## Checklist:
- [x] Tests added/updated.
- [ ] Documentation updated.
- [ ] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/power-ops-sdk/blob/master/CHANGELOG.md).
- [x] Version bumped. If triggering a new release is desired, bump the version number in [pyproject.toml](https://github.com/cognitedata/power-ops-sdk/blob/master/pyproject.toml) per [semantic versioning](https://semver.org/).

### CODE REVIEW

**Comment by katrilh** [comment]
Pytests failing, relationships to generators missing in approval test

File: cognite/powerops/resync/to_models/to_production_model.py
Comment (doctrino) [inline cognite/powerops/resync/to_models/to_production_model.py:192]: Here is the issue :)

**Review by doctrino — COMMENTED** [review]
I realize when reading through that is is probably possible to write the `from_asset` in a generic way. But lest get this one merged first. Remember to update changelog + version before merging.

**Review by doctrino — APPROVED** [review]

**Pull request merged**
